python/cudf/cudf/io/avro.py

```diff
@@ -10,7 +8,6 @@
 @ioutils.doc_read_avro()
```

Comment:
We can drop the documentation for `engine` parameter here too: https://github.com/rapidsai/cudf/blob/branch-23.02/python/cudf/cudf/utils/ioutils.py#L47